src/pages/index.js

```diff
@@ -33,40 +60,94 @@ const cardFormValidator = new FormValidator(
   constants.cardFormElement
 );
 
-editFormValidator.enableValidation();
-cardFormValidator.enableValidation();
-
-const cardCreator = new Section(
-  {
-    items: constants.initialCards,
-    renderer: createCard,
-  },
-  ".cards__list"
+const avatarFormValidator = new FormValidator(
+  constants.validationSettings,
+  constants.avatarFormElement
 );
 
-cardCreator.renderItems();
+editFormValidator.enableValidation();
+cardFormValidator.enableValidation();
+avatarFormValidator.enableValidation();
 
 const userProfileInfo = new UserInfo({
   profileName: ".profile__title",
   profileJob: ".profile__subtitle",
+  profileAvatar: ".profile__image",
 });
 
+api
+  .getUserInfo()
+  .then((data) => {
+    console.log(">>USER DATA:", data);
+    userProfileInfo.setUserInfo({
+      profileName: data.name,
+      profileJob: data.about,
+      profileAvatar: data.avatar,
+    });
+  })
+  .catch((err) => console.error(err));
+
 function handleProfileSubmit(formValues) {
-  const profileName = formValues.title;
-  const profileJob = formValues.subtitle;
-  userProfileInfo.setUserInfo({ profileName, profileJob });
-  profilePopup.close();
+  profilePopup.setLoadingState(true);
+
+  api
+    .updateUserInfo({ name: formValues.title, about: formValues.subtitle })
+    .then((updatedData) => {
+      userProfileInfo.setUserInfo({
+        profileName: updatedData.name,
+        profileJob: updatedData.about,
+      });
+      profilePopup.close();
+    })
+    .catch((err) => console.error(err))
+    .finally(() => {
+      profilePopup.setLoadingState(false);
+    });
 }
 
 function handleAddCardSubmit(formValues) {
-  const name = formValues.title;
-  const link = formValues.url;
-  const data = { name, link };
+  cardPopup.setLoadingState(true);
+
+  const { title, url } = formValues;
+  api
+    .addNewCard({ name: title, link: url })
+    .then((newCard) => {
+      cardCreator.addItem(getCardElement(newCard));
+      cardPopup.close();
```

Comment:
* You should **clear the inputs** after a successful adding of a new card to let the user add the 2nd one again without having to remove the old data manually.  You should do that in a `.then` block


You'll have `resetForm` for resetting the form

```js
    .then((newCard) => {
      cardCreator.addItem(getCardElement(newCard));
      cardPopup.close();
      cardPopup.resetForm()  // please, use it here to clear all inputs
      cardFormValidator.disableButton();
    })
```